types/moment-holiday/moment-holiday-tests.ts

```diff
@@ -0,0 +1,126 @@
+import * as moment from 'moment';
```

Comment:
Can you use the `import moment = require('moment')` syntax, and explicitly import `moment-holiday` as well?

Reply:
Sure thing!